t/16_create_audience_for_uploading.t

```diff
@@ -1,6 +1,6 @@
 use strict;
 use warnings;
-use Test::More;
+use Test2::V0;
```

Comment:
👍 BTW, with the introduction of `Test2::V0`, `string` and `warnings` are also implied and can be removed.

Reply:
I removed `strict` and `warnings` [here](https://github.com/line/line-bot-sdk-perl/pull/162/files#diff-d7d28a50713ee0848c97f5eec121410be72a87319e13b46e68c36a2f262a6d40L1-L2)

thank you for commenting :)